CHANGELOG.md

```diff
@@ -1,10 +1,87 @@
 # Compliant Kubernetes changelog
 <!-- BEGIN TOC -->
+- [v0.8.0](#v080---2020-12-11)
 - [v0.7.0](#v070---2020-11-09)
 - [v0.6.0](#v060---2020-10-16)
 - [v0.5.0](#v050---2020-08-06)
 <!-- END TOC -->
 
+-------------------------------------------------
+
+## v0.8.0 - 2020-12-11
```

Comment:
Shouldn't this be the changelog for 0.9.0?

Reply:
Fixed.